.github/workflows/benchmark-pipeline-test.yml

```diff
@@ -17,8 +14,3 @@ jobs:
           version: "0.15.4"
           args: benchmark-pipeline-test
               --source='.'
-              --cncf-project='falco'
-              --config='modern-ebpf'
-              --version='0.39.2'
-              --benchmark-job-url='https://raw.githubusercontent.com/falcosecurity/cncf-green-review-testing/2551137b1a09bd0594f76b09e82e08c98f95efd3/benchmark-tests/falco-benchmark-tests.yaml'
-              --benchmark-job-duration-mins=2
```

Comment:
These params are duplicated in the BenchmarkPipelineTest job we run when calling `make test`
https://github.com/cncf-tags/green-reviews-tooling/blob/56fedac11a85ea3231c02568693832b9c245b7e5/main.go#L39-L57

I think its best to use the default values there and not duplicate here as they will get out of sync.